CHANGES.md

```diff
@@ -4335,7 +4335,7 @@ OpenSSL 1.1.0
 
    *Matt Caswell*
 
- * Excessive allocation of memory in tls_get_message_header() and
+ * Excessive allocation of memory in tls_get_message_body() and
```

Comment:
Ahem, this should be kept as is.

Please squash the commits with `git rebase -i master` and drop all the changes in CHANGES.md. There should not be any.

Reply:
Okay.

Reply:
Hi @t8m I've encountered significant issues in the ssl3_to_tls_message_comments branch, leading to a complex situation. Considering this, I'm contemplating creating a new branch and reapplying the changes to ensure a cleaner and more manageable state. Will that be fine?